_i18n/es/resources/user-guides/view_only.md

```diff
@@ -1,34 +1,34 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-A view-only wallet can only see which incoming transactions belong to you. It can not spend any of your Monero, in fact it can't even see outgoing transactions from this wallet. This makes view-only wallets particularly interesting for
+{% include disclaimer.html translated="true" version=page.version %}
+Un monedero de visualización solamente puede ver qué transacciones de entrada te pertenecen. No puede utilizar nada de tu Monero, de hecho ni siquiera puede ver las transacciones de salida del monedero. Esto hace a los monederos de sólo visualización interesantes para
 
-* Developers writing libraries to validate payments
-* End users validating incoming transactions to cold wallets
+* Desarrolladores escribiendo librerías para validar pagos
+* Usuarios finales validando transacciones de entrada para monederos de almacenamiento en frío
 
-### Creating A View-Only Wallet
+### Creando un Monedero de Visualización
 
-You can create a view-only wallet from any existing wallet.
+Puedes crear un monedero de visualización de cualquier monedero existente.
 
 #### CLI
 
-Open an existing wallet or create a new one using `monero-wallet-cli`. In the wallet, type `address` and `viewkey` to display the wallet's address and secret viewkey. Type `exit` to close the wallet.
+Abre un monedero existente o crea uno nuevo usando `monero-wallet-cli`. En el monedero, escribe `address` y `viewkey` para mostrar la dirección del monedero y la clave secreta de visualización. Escribe `exit` para cerrar el monedero.
 
-Next, create your view-only wallet by typing `monero-wallet-cli --generate-from-view-key wallet-name`. The last argument will be your new wallet's file name. You will be prompted for `Standard address` and `View key` by the wallet. Paste in your original wallet's address and secret view key. Next, enter and confirm a password for your new wallet and you're done.
+A continuación, crea tu monedero de visualización escribiendo `monero-wallet-cli --generate-from-view-key wallet-name`. El último argumento será el nuevo nombre de tu monedero. Se te preguntará por un `Standard address` y `View key` por el monedero. Pega la dirección original de tu monedero y la clave secreta de visualización. A continuación, ingresa y confirma una contraseña para tu nuevo monedero y estarás listo.
 
 #### GUI
 
-Open an existing wallet or create a new one using `monero-wallet-gui`. In the wallet, go to the `settings` page:
+Abre un monedero existente o crea uno nuevo usando `monero-wallet-gui`. En el ejemplo, ve a la página de `ajustes`
```

Comment:
I opened an discussion in the [spanish terminology - issue #22](https://github.com/monero-ecosystem/monero-translations/issues/22)  for `settings` .